lfshttp/standalone/standalone.go

```diff
@@ -0,0 +1,228 @@
+package standalone
+
+import (
+	"bufio"
+	"encoding/json"
+	"fmt"
+	"net/url"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/git-lfs/git-lfs/config"
+	"github.com/git-lfs/git-lfs/errors"
+	"github.com/git-lfs/git-lfs/lfs"
+	"github.com/git-lfs/git-lfs/lfsapi"
+	"github.com/git-lfs/git-lfs/subprocess"
+	"github.com/rubyist/tracerx"
+)
+
+type inputMessage struct {
+	Event     string `json:"event"`
+	Operation string `json:"operation"`
+	Remote    string `json:"remote"`
+	Oid       string `json:"oid"`
+	Size      int64  `json:"size"`
+	Path      string `json:"path"`
+}
+
+type errorMessage struct {
+	Message string `json:"message"`
+}
+
+type completeMessage struct {
+	Event string        `json:"event"`
+	Oid   string        `json:"oid"`
+	Path  string        `json:"path,omitempty"`
+	Error *errorMessage `json:"error,omitempty"`
+}
+
+type fileHandler struct {
+	remotePath   string
+	remoteConfig *config.Configuration
+	output       *os.File
+	config       *config.Configuration
+}
+
+func fileUrlFromRemote(cfg *config.Configuration, name string, direction string) (*url.URL, error) {
```

Comment:
I think it would be worth adding some more detailed comments here on what each of the functions are doing. Since this is at least partially intended as an example, that'll help make it easier for people to use it as such

Reply:
That sounds like a good idea.